2.train-VAE/scripts/nbconverted/beta_vae_training.py

```diff
@@ -29,56 +28,117 @@
 # In[2]:
 
 
-# load the data 
-data_directory = "../0.data-download/data/"
-dfs = load_data(data_directory, adult_or_pediatric = "all")
-dependency_df = dfs[1]
-sample_df = dfs[0]
+# load the data
+data_directory = pathlib.Path("../0.data-download/data")
+dfs = load_train_test_data(data_directory, train_or_test = "all")
+train_init = dfs[0]
+test_init = dfs[1]
+gene_stats = dfs[2]
 
 
 # In[3]:
 
 
-dependency_df
+# drop the string values
+train_df = train_init.drop(columns= ["DepMap_ID", "age_and_sex"])
+test_df = test_init.drop(columns= ["DepMap_ID", "age_and_sex"])
 
 
 # In[4]:
 
 
-dependency_df1 = dependency_df.drop(axis=0, columns= "DepMap_ID")
-dependency_df1.reset_index(drop=True, inplace=True)
-print(dependency_df1.shape)
-dependency_df1.head(3)
+# subsetting the genes 
+# create dataframe containing the 1000 genes with the largest variances and their corresponding gene label and extract the gene labels
+largest_var_df = gene_stats.nlargest(1000, "variance")
+gene_list = largest_var_df["gene_ID"].tolist()
+gene_list
+
+# create new training and testing dataframes that contain only the corresponding genes
+subset_train_df = train_df.filter(gene_list, axis = 1)
+subset_test_df = test_df.filter(gene_list, axis = 1)
 
 
 # In[5]:
 
 
-encoder_architecture = [250]
-decoder_architecture = [250]
+print(subset_train_df.shape)
+subset_train_df.head(3)
 
 
 # In[6]:
 
 
+print(subset_test_df.shape)
+subset_test_df.head(3)
+
+
+# In[7]:
+
+
+# scale the data
+def absolute_maximum_scale(series):
```

Comment:
oh interesting, I don't remember talking about this. What is this normalization?

Reply:
this is an absolute scaling method that sets all values to fall between 1 and -1

Reply:
gotcha - this normalization is not typical. Can you add a reference to the code to describe why you chose this method?

Reply:
this was a safety normalization to ensure that there were no outlying values in the data set. the actual changes to the data were minimal. Im beggining to think that this extra normalization code is a bit arbitrary